teambit.bvm/link/link.ts

```diff
@@ -25,29 +26,37 @@ export type GeneratedLink = {
   target: string
 }
 
-export async function linkAll(): Promise<LinkResult[]>{
+export async function linkAll(opts: { updatePath?: boolean }): Promise<LinkResult[]>{
   const links = config.getLinks();
   const defaultLinkVersion = config.getDefaultLinkVersion();
   const localLatest = (await listLocal()).latest();
   const promises = Object.entries(links).map(([linkName, version]) => {
-    return linkOne(linkName, version, false);
+    return linkOne(linkName, version, { addToConfig: false, updatePath: opts.updatePath });
   });
   if (!defaultLinkVersion && localLatest){
     const defaultLinkName = config.getDefaultLinkName();
-    promises.push(linkOne(defaultLinkName, localLatest.version, true))
+    promises.push(linkOne(defaultLinkName, localLatest.version, { addToConfig: true, updatePath: opts.updatePath }))
   }
   return Promise.all(promises);
 }
 
+export interface LinkOptions {
+  addToConfig?: boolean
+  updatePath?: boolean
```

Comment:
I would change it to maybe to something that implies that it will updatePath only in case of missing.
(maybe we will want soon to have an option to "force" the update path, even if it's not missing) in such case it will be confusing.

Reply:
`initPath` then?

Reply:
@zkochan sound fine to me

Reply:
done